Add reference to dozer-spring-boot-starter

kazuki43zoo opened the pull request:
The [Dozer](https://github.com/DozerMapper/dozer) is the Java Bean mapper. Since the Dozer 6.2.0(current latest version), the dozer-spring-boot-starter module was added. Hence; I will introduce the it as community starter.

bclozel commented:
Looking at the Dozer starter, it seems it is only compatible with Spring Boot 1.5 - is that the case?
In the meantime, we're thinking about pruning the current list from starters incompatible with Spring Boot 2.x. This should be done on the master/2.0.x branches.

Depending on the outcome of that discussion in #13510, this PR could be declined.

kazuki43zoo commented:
@bclozel @snicoll Thanks for your replying!
I will consult it with a maintainer of dozer.

kazuki43zoo commented:
@bclozel 
I've added the compatibility checker project on my account. See https://github.com/kazuki43zoo/dozer-spring-boot-compatibility-checker. The dozer-spring-boot-starter 6.2.0 depends on Spring Boot 1.5.x, but it works fine on Spring Boot 1.4.x - 2.1.x.

@snicoll 
I've fixed your comment.
